### PR TITLE
Past Events Update (April 2023)

### LINKED ISSUE #6: Past Events Update (April 2023)
Update the past events to include the April 2023 presentation information.

### PR DESCRIPTION
**Pull request by Jeymz**
Added April 2023 information and presentation artifact.

closes #6

### CODE REVIEW

File: tab_pastevents.md [hunk 1]
```diff
@@ -12,6 +12,16 @@ tags: cincinnati
 ## Past Events
 <hr>
 
+## April 2023 Meeting
+
+### Do AppSec Better
+**Discussion Abstract:** In this talk Jay will take us through his experience at the recent OWASP SnowFROC conference and share his main takeaways from the event.
+
+**Speaker Biography** Jay Simmons is a Senior AppSec Analyst for Great American and Security Champion program administrator. He's been working in the AppSec field for over 2 years and was previously a developer.
```
Comment (szh) [inline tab_pastevents.md]:
```suggestion
**Speaker Biography:** Jay Simmons is a Senior AppSec Analyst for Great American and Security Champion program administrator. He's been working in the AppSec field for over 2 years and was previously a developer.
```

File: tab_pastevents.md [hunk 2]
```diff
@@ -12,6 +12,16 @@ tags: cincinnati
...
+
+**Presentation**
```
Comment (szh) [inline tab_pastevents.md]:
```suggestion
**Presentation:**
```

**Review by szh — CHANGES_REQUESTED** [review]
Thank you for adding this! Two missing colons, otherwise LGTM!

**Comment by Jeymz** [comment]
> Thank you for adding this! Two missing colons, otherwise LGTM!

Committed to request. Should be good to go now.

**Review by szh — APPROVED** [review]
LGTM

**Pull request merged**